extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -189,21 +189,25 @@ public final class HibernateOrmProcessor {
     void registerHibernateOrmMetadataForCoreDialects(
             BuildProducer<DatabaseKindDialectBuildItem> producer) {
         producer.produce(new DatabaseKindDialectBuildItem(DatabaseKind.DB2,
-                "org.hibernate.dialect.DB297Dialect"));
+                "org.hibernate.dialect.DB2Dialect"));
```

Comment:
LGTM, the default (min) version is now 10.5:

https://github.com/hibernate/hibernate-orm/blob/2019fca0cb405cf60fcab6099e850be9dabd0a74/hibernate-core/src/main/java/org/hibernate/dialect/DB2Dialect.java#L113